EventForwarding/EventForwarding_CZ_regulator.md

```diff
@@ -0,0 +1,146 @@
+## Teplotní regulátor
+Reguluje teplotu radiátorů pomocí termostatických hlavic na základě dat měřených teploměrem. Standardně zasílá události s naměřenou teplotu a vlhkostí okolního prostředí.
+
+Každou minutu měří teplotu a vlhkost. Po X měření provede výpočet průměrné hodnoty a odešle událost `measured-humidity-temperature`.
+
+> DeviceType: temperature-regulator
```

Comment:
Ja bych tady nekde hned nazacatku vysvetlil to ze se ten regulator sklada z X fyzickych zarizeni kterymi jsou hlavice a teplomer.

Reply:
ok